help/sources/tutorials/api/create/crm/sugarcrm-events.md

```diff
@@ -0,0 +1,662 @@
+---
+keywords: Experience Platform;home;popular topics;sources;connectors;source connectors;sources sdk;sdk;SDK
+solution: Experience Platform
+title: Create a source connection and dataflow for SugarCRM Events using the Flow Service API
+topic-legacy: tutorial
+description: Learn how to connect Adobe Experience Platform to SugarCRM Events using the Flow Service API.
+hide: true
+hidefromtoc: true
+---
+# Create a source connection and dataflow for [!DNL SugarCRM Events] using the Flow Service API
+
+[[!DNL SugarCRM]](https://www.sugarcrm.com/) is a customer relationship management (CRM) system. [!DNL SugarCRM]'s functionality includes sales-force automation, marketing campaigns, customer support, collaboration, Mobile CRM, Social CRM and reporting.
+
+To retrieve event data setup within [!DNL SugarCRM], this Adobe Experience Platform [source](https://experienceleague.adobe.com/docs/experience-platform/sources/home.html?lang=en) connector leverages the [!DNL SugarCRM] [Events API](https://market.apidocs.sugarcrm.com/#516ec3b1-8e70-43d4-8bf2-38a2ae74c0a5).
+
+After you bring that data to Experience Platform, you can then execute any analytics.
+
+[!DNL SugarCRM Events] uses bearer tokens as an authentication mechanism to communicate with the [!DNL SugarCRM] Events API.
+
+## Prerequisites
+
+The first step in creating a [!DNL SugarCRM Events] source connection is to ensure that you have the below :
+1. A [!DNL SugarMarket] *(SugarCRM’s product for Marketing Automation)* account.
+1. Additionally, a unique API username and user account separate from any user account associated with the marketing or sales process; having API Access permissions in order to access the API. The process to obtain / set up the account is documented on the [[!DNL SugarMarket RESTFUL API (sugarcrm.com)]](https://market.apidocs.sugarcrm.com/#intro) page.
```

Comment:
Let's remove this. I will make a separate page on experience league to contain the pre-reqs for SugarCRM. keep the one for accounts though

Reply:
Removed from the API
  ```help/sources/tutorials/api/create/crm/sugarcrm-events.md```